Automattic/hidden-posts
# 📦 NEW: Prepare plugin for build tools

Fixes #21 

**Change**

Currently, the plugin is not prepared for build tools as it does not have a `composer.json` and a `package.json` file. This PR aims to prepare the plugin to install build tools in the future. This PR also contain `.editorconfig`, `.gitignore` and `phpcs.xml`.

**Steps to test**

1. Check out this plugin
2. Check if the following files are available in the root folder
    - `composer.json`
    -  `composer.lock`
    -  `package.json`
    - `.editorconfig`
    - `.gitignore`
    - `phpcs.xml`

**Note**

Initially, I also planned on adding [PHPUnit](https://phpunit.de/) and [Cypress](https://www.cypress.io/). To keep this PR lean, I decided creating separate issues and PRs for that.
